### ISSUE TITLE
[Feature Request] Document encryption

### ISSUE THREAD

**Issue by IWaster** [rust-lang/mdBook#1782]
Can we add password encryption for a single document or a single directory later? Only after entering the password can you read it

**Comment by nassersaazi** [comment]
This would be such a nice to have :)

**Comment by ehuss** [comment]
I'm going to close as this probably isn't a feature we would accept.  Perhaps in the future it might be feasible to implement via some kind of extension.
